src/scripts/meza.py

```diff
@@ -1174,7 +1174,7 @@ def prompt(varname,default=False):
 	else:
 		# If no default, keep asking until user supplies a value
 		while (not value):
-			value = raw_input( input_msg )
```

Comment:
The raw_input() function which is available in python 2 is used to take the input entered by the user. The raw_input() function explicitly converts the entered data into a string and returns the value. The raw_input() can be used only in python 2.